logging/sentry/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@rango-dev/logging-sentry",
+  "version": "0.1.0",
+  "license": "MIT",
+  "type": "module",
+  "source": "./src/index.ts",
+  "main": "./dist/index.js",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "dependencies": {
+    "@rango-dev/logging-types": "^0.1.0"
+  },
+  "devDependencies": {
+    "@sentry/browser": "^7.102.1"
+  },
```

Comment:
I think sentry should be a peer dependency or dependency for this package not a dev dependency.

Reply:
We are only importing some types from the package which is only needed for development and will be omitted during build. 
It's not peer dependency as well, because the package don't need sentry to work. the instance will be passed when app is going to initialize the layer.